doc/CMakeLists.txt

```diff
@@ -44,6 +44,20 @@ if(DOXYGEN_FOUND)
     # configure doxygen target 
     set(RADIUM_DOC_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/)
     set(RADIUM_MAIN_DOC_FILE ${CMAKE_CURRENT_SOURCE_DIR}/main.md)
+
```

Comment:
Could you do that only if `GIT_CHANGESET` does not exists ?

Reply:
Yes, but it works in both cases (and I found adding a IF here could be misleading ...)
So if you insists I'll do it ;)

Reply:
Thinking about it, I think it is better to keep the computation of this value in this file, so I am fine with the current version.